src/modes/single/SingleLocation.svelte

```diff
@@ -11,20 +11,29 @@
   } from '../../stores';
   import { sensorList } from '../../stores/constants';
   import SensorCard from './SensorCard.svelte';
-  import throttle from 'lodash-es/throttle';
+  import debounce from 'lodash-es/throttle';
   import { selectionColors } from '../../theme';
   import { resolveHighlightedTimeValue } from '../overview/vegaSpec';
 
   let highlightTimeValue = null;
 
-  const throttled = throttle((value) => {
-    highlightTimeValue = value;
-  }, 25);
+  // Each mouse move across the chart results in a mouseout for previous highlight
+  // date immediately followed by mouseover for the next date, if any.  We need to
+  // ignore the first event unless there is no second.  A very short debounce time
+  // achieves that goal, at least most of the time.
+  const debouncedHighlightTime = debounce(
+    (value) => {
+      console.info('highlighted valuu', value);
```

Comment:
can you remove the logging statement, please

Reply:
Done, but forgot to push it.

Reply:
please push it

Reply:
OK, I thought I did it, but must have been in another branch.  Did it again.